packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -393,7 +393,7 @@ export async function* OpenAIChatModel(
     if (delta.content) {
       currentMessage.content = currentMessage.content ?? '';
       currentMessage.content += delta.content;
-      yield currentMessage.content;
+      yield delta.content;
     }
     if (delta.function_call) {
```

Comment:
Is it weird that we're never `yield`ing the function call?

Reply:
It happens to be harmless but I agree that it's weird -- fixed.